### ISSUE TITLE
How do I further debug: Unknown Column Type 

### ISSUE THREAD

**Issue by TheServerAsterisk** [blackbeam/rust-mysql-simple#88]
The error seems to be occurring at Column::from_payload 

Stack Trace 
```
stack backtrace:
   1:     0x5599b52f9c4f - std::sys::backtrace::tracing::imp::write::h6f1d53a70916b90d
   2:     0x5599b52ff3cd - std::panicking::default_hook::{{closure}}::h137e876f7d3b5850
   3:     0x5599b52fde7a - std::panicking::default_hook::h0ac3811ec7cee78c
   4:     0x5599b52fe478 - std::panicking::rust_panic_with_hook::hc303199e04562edf
   5:     0x5599b526953f - std::panicking::begin_panic::h341b039f84d0b176
   6:     0x5599b5279bc1 - mysql::conn::Column::from_payload::h09591b330bffddd4
   7:     0x5599b5281ed2 - mysql::conn::Conn::_prepare::hdca0d84f6439e67b
   8:     0x5599b514ef0a - mysql::conn::pool::PooledConn::pooled_prep_exec::h26180efa877ffa59
   9:     0x5599b5152066 - mysql::conn::pool::Pool::prep_exec::hb2a497d205d93bd3
  10:     0x5599b51b554b - <api::controller::dsihd::QueryRoot as juniper::types::base::GraphQLType>::resolve_field::h2cc35af3d271cf73
  11:     0x5599b50d2923 - juniper::types::base::resolve_selection_set_into::hcaa9c60086a37ef7
  12:     0x5599b5093ce9 - <juniper::integrations::iron_handlers::GraphQLHandler<'a, CtxFactory, Query, Mutation, CtxT>>::execute::h1e9209e9339ef3ce
  13:     0x5599b509e888 - <juniper::integrations::iron_handlers::GraphQLHandler<'a, CtxFactory, Query, Mutation, CtxT> as iron::middleware::Handler>::handle::h42e58e1bacb84f03
  14:     0x5599b524ad40 - <Box<iron::middleware::Handler + 'static> as iron::middleware::Handler>::handle::h5c664afc2d81d9ce
  15:     0x5599b50e7486 - <mount::mount::Mount as iron::middleware::Handler>::handle::h2939ed117ba0a0f6
  16:     0x5599b524a3b5 - iron::middleware::Chain::continue_from_handler::hd0ff7a6a40a9f4c2
  17:     0x5599b5249b5a - <iron::middleware::Chain as iron::middleware::Handler>::handle::h7d364e4f837780e3
  18:     0x5599b50bf9e3 - <iron::iron::Iron<H> as hyper::server::Handler>::handle::hc53870f547cfd778
  19:     0x5599b50aaf83 - std::panicking::try::do_call::h31150abe3c16767a
  20:     0x5599b5306e96 - __rust_maybe_catch_panic
  21:     0x5599b50b9129 - <F as alloc::boxed::FnBox<A>>::call_box::h395cde831ec4edba
  22:     0x5599b52fcda0 - std::sys::thread::Thread::new::thread_start::h759e10bc4abc7e72
  23:     0x7f5dede666c9 - start_thread
  24:     0x7f5ded989f7e - clone
  25:                0x0 - <unknown>
```

The data types that the querying is of type varchar and of type json.  I am going to guess it is the json column that is making it panic. 

However, the query would be: 

```
SELECT
    content_type,
    content_data
FROM
     units_playlists
 WHERE
      uuid = :uuid  # where :uuid will could be evaluated to sdkljsdf8902345rjklsdfu904tr
```

I confirmed the query works via the mysql client. Any idea on how to further debug this would be great. We are currently using mysql 5.7.16.

**Comment by blackbeam** [comment]
Json support is implemented in e2345d4.
Could you please test it for convenience?
This is easy, you just need to alter your Cargo.toml for this:

```toml
[dependencies.mysql]
git = "https://github.com/blackbeam/rust-mysql-simple.git"
```

### Docs

Now `rustc_serialize::json::Json` could be passed as a parameter and retrieved from row directly.
Also `T: Encodable` could be passed as a parameter if wrapped in `Serialized`
and `T: Decodable` could be retrieved from row as `Unserialized`.

Here is some code for reference:

```rust
#[test]
fn should_handle_json_columns() {
    use rustc_serialize::json::Json;
    use value::{Serialized, Unserialized};

    #[derive(RustcDecodable, RustcEncodable, Debug, Eq, PartialEq)]
    struct DecTest {
        foo: String, 
        quux: (u64, String),
    }

    let decodable = DecTest {
        foo: "bar".into(),
        quux: (42, "hello".into()),
    };

    let mut conn = Conn::new(get_opts()).unwrap();
    conn.query("CREATE TEMPORARY TABLE x.tbl(a VARCHAR(32), b JSON)").unwrap();
    conn.prep_exec(
        r#"INSERT INTO x.tbl VALUES ('hello', ?)"#,
        (Serialized(&decodable),)
    ).unwrap();

    let row = conn.first("SELECT a, b FROM x.tbl").unwrap().unwrap();
    let (a, b): (String, Json) = from_row(row);
    assert_eq!((a, b), ("hello".into(), Json::from_str(r#"{"foo": "bar", "quux": [42, "hello"]}"#).unwrap()));


    let row = conn.first_exec("SELECT a, b FROM x.tbl WHERE a = ?", ("hello",)).unwrap().unwrap();
    let (a, Unserialized(b)) = from_row(row);
    assert_eq!((a, b), (String::from("hello"), decodable));
}
```

I willl publish this after your response.

This will also close #87.

**Comment by TheServerAsterisk** [comment]
Reference code tested successfully. 👍

**Comment by blackbeam** [comment]
Fixed in v9.0.0.
